src/test/resources/alinaTinca/suiteTema8Desktop.xml

```diff
@@ -0,0 +1,16 @@
+<suite name="Homework8-Desktop Test Suite" verbose="1" >
```

Comment:
Why are suite.xml and group.xml files empty? Please refer to #39 and put the content back inside them. Thanks!